tests/test_hoomd.py

```diff
@@ -496,3 +496,33 @@ def test_state(tmp_path):
 
         numpy.testing.assert_array_equal(s.state['hpmc/convex_polyhedron/N'], snap1.state['hpmc/convex_polyhedron/N']);
         numpy.testing.assert_array_equal(s.state['hpmc/convex_polyhedron/vertices'], snap1.state['hpmc/convex_polyhedron/vertices']);
+
+def test_log(tmp_path):
+    snap0 = gsd.hoomd.Snapshot();
+
+    snap0.log['particles/net_force'] = [[1,2,3],[4,5,6]]
+    snap0.log['value/potential_energy'] = [10]
```

Comment:
Following @bdice's previous comments, perhaps we can use NumPy arrays here too. Same applies below.

Reply:
Feel free to make the change if you want. The specification assigns no particular data types for any particular chunk names. This unit test correctly validates that data, as defined by implicit numpy conversion rules, survives the round trip from write to read.

Reply:
In that case we can leave it as is.